site/docs/v1/tech/account-management/totp-two-factor.adoc

```diff
@@ -0,0 +1,37 @@
+---
+layout: doc
+title: Account Management - Add TOTP Two Factor
+description: Managing Accounts - Add TOTP Two Factor
+---
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Add a Two Factor Method - TOTP
+
+The authenticator is the only service enabled by default. To fully enable:
```

Comment:
This MFA method is enabled by default on every tenant.

To enable this method for a particular user: